models/tasks.js

```diff
@@ -284,6 +284,12 @@ const overdueTasks = async (overDueTasks) => {
     throw err;
   }
 };
+
+const generateTaskNo = async () => {
+  const snapshot = await tasksModel.count().get();
```

Comment:
Is this working for you?

Reply:
Yes, it is. Attached a screenshot of the firestore that created a task number when we hit a post request to create new task

Reply:
Okay. I have tried using the same but got an error count that is not a function. So just wanted to check if it's working.

Reply:
count function requires latest version of firebase SDK, make sure you have an updated one